packages/features/data-table/components/filters/AddFilterButton.tsx

```diff
@@ -17,17 +16,10 @@ import { useDataTable, useFilterableColumns } from "../../hooks";
 export interface AddFilterButtonProps<TData> {
   table: Table<TData>;
   variant?: "base" | "sm";
-  hideWhenFilterApplied?: boolean;
-  showWhenFilterApplied?: boolean;
```

Comment:
Instead of these props, we handle it at the caller level. See the "FilterBar" component below.